README.md

```diff
@@ -549,9 +579,10 @@ In this extremely short tutorial we are going to mimic the simplest flow: regist
 
 ## Customization
 
-If you need to override some `djoser` behaviour, you could define your custom view/serializer.
+If you need a use custom serializer, use `DJOSER[SERIALIZERS]`.
+And if you need to override some `djoser` behaviour, you could define your custom view/serializer.
```

Comment:
Or even:
If you need to customize any serializer behavior you can use the DJOSER['SERIALIZERS'] setting to use your own serializer classes in the built-in views. Or if you need to completely change the default djoser behaviour, you can always override djoser views with your own custom ones.